### PR TITLE
Add purl information to SPDX

### PR DESCRIPTION
**Pull request by ivanayov**
This change adds purl data to all packages listed

Addresses #1206

Signed-off-by: Ivana Atanasova <iyovcheva@vmware.com>

### CODE REVIEW

File: tern/classes/image_layer.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 #
-# Copyright (c) 2017-2021 VMware, Inc. All Rights Reserved.
+# Copyright (c) 2017-2022 VMware, Inc. All Rights Reserved.
```
Comment (rnjudge) [inline tern/classes/image_layer.py:3]: It is 2023 already! :)

Comment (ivanayov) [reply tern/classes/image_layer.py]: Thanks for noticing 😄

**Comment by ivanayov** [comment]
Testing the failure of `tern report -f spdxjson -i photon:3.0 -o spdx.json` locally the `-o` flag doesn't generate any output. Same when building from the main branch. Seems like this fails in the CI as well. 

Looks like not related to the PR changes, but probably a bug.

**Comment by rnjudge** [comment]
Finally getting around to testing this... The CI issue is related to SPDX document validation:
```
subprocess.CalledProcessError: Command 'tern report -f spdxjson -i photon:3.0 -o spdx.json && java -jar tools-java/target/tools-java-*-jar-with-dependencies.jar Verify spdx.json' returned non-zero exit status 1.
```

This error means that the SPDX JSON resulting from these changes is not validating properly.

**Comment by ivanayov** [comment]
Closing as it's addressed in a follow-up.

**Pull request closed without merging**